x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/common/card_content_panel.tsx

```diff
@@ -11,7 +11,7 @@ export const OnboardingCardContentPanel = React.memo<PropsWithChildren<EuiPanelP
   ({ children, ...panelProps }) => {
     return (
       <EuiPanel hasShadow={false} hasBorder={false} paddingSize="m">
-        <EuiPanel {...panelProps} hasShadow={false} paddingSize="l">
+        <EuiPanel {...panelProps} hasShadow={false}>
```

Comment:
I think that what we want here is `style={{ paddingTop: 0 }}` instead of removing `paddingSize="l"` the rest of the paddings are fine.